### ISSUE TITLE
TypeScript: type literal members should be separated by semicolons

### ISSUE THREAD

**Issue by clebert** [prettier/prettier#1896]
Input:

```js
declare module 'selenium-webdriver' {
  export const until: {
    ableToSwitchToFrame(frame: number | WebElement | By): Condition<boolean>;
    alertIsPresent(): Condition<Alert>;
  };
}
```

Output:

```js
declare module 'selenium-webdriver' {
  export const until: {
    ableToSwitchToFrame(frame: number | WebElement | By): Condition<boolean>,
    alertIsPresent(): Condition<Alert>,
  };
}
```

The TSLint rule `type-literal-delimiter` checks that type literal members are separated by semicolons.

**Comment by pelotom** [comment]
Wow, I just came here to file this bug and you created it literally a minute beforehand 😆

Just to add to this, type members should be separated by semicolons, but if semicolons are disabled and members are on separate lines there should be no semicolons (or commas). E.g.,

```ts
type Foo = {
  x: number
  y: boolean
}
```

**Comment by vjeux** [comment]
Likely the same issue as https://github.com/prettier/prettier/issues/1879

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!
